hosts_new

```diff
@@ -0,0 +1 @@
+192.168.121.[30:130]
```

Comment:
I was using this file to test passing in rho like host ranges to ansible. I now renamed both the host files to eg_hosts_1 and eg_hosts_2 to indicate that I use those hosts for development on my machine.